container_images/daisy-builder/main.sh

```diff
@@ -20,6 +20,20 @@ ZONE="$2"
 WORKFLOW_FILE="$3"      # Workflow to run
 GCS_OUTPUT_BUCKET="$4"  # Destination for artifacts
 
+function GENERATE_NEW_VERSION() {
+  VERSION_GENERATE_CMD="/versiongenerator --token-file-path=${GITHUB_ACCESS_TOKEN} --org=${REPO_OWNER} --repo=${REPO_NAME}"
```

Comment:
why store as a variable just to use in next line?

Reply:
used the variable to debug env variables. i have a echo in my local repo to verify that the env variables are correctly set.

Reply:
kk, let's just prune from the PR

Reply:
done